src/goto-instrument/cover.cpp

```diff
@@ -184,11 +176,13 @@ void parse_cover_options(const cmdlinet &cmdline, optionst &options)
 
 /// Build data structures controlling coverage from command-line options.
 /// \param options: command-line options
+/// \param main_symbol: symbol of the user-specified main program function
 /// \param symbol_table: global symbol table
 /// \param message_handler: used to log incorrect option specifications
 /// \return a cover_configt on success, or null otherwise.
 std::unique_ptr<cover_configt> get_cover_config(
   const optionst &options,
+  const optionalt<symbolt> &main_symbol,
```

Comment:
Is this actually used?

Reply:
Yes it is eventually, I did a bad rebase and it was not used in the particular commit.